Translated/Dialog/np_npc_00cmn_reboot.csv

```diff
@@ -0,0 +1,113 @@
+WINDOW_TITLE#0,"""What?"""
```

Comment:
We didn't translate this file for a reason. We know it'll likely be used in the future, however until then do not translate this.